python/ray/rllib/optimizers/util.py

```diff
@@ -1,33 +0,0 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-import time
-
-# TODO(ekl) move this to a common location
-from ray.rllib.ppo.filter import RunningStat
-
-
-class TimerStat(RunningStat):
-    """A running stat for conveniently logging the duration of a code block.
```

Comment:
Isn't this class still used?

Reply:
I think this was duplicate; there's a `timer.py` in utils (that I don't think I touched)